src/drc/api/tests/test_validation.py

```diff
@@ -219,6 +224,140 @@ def test_inhoud_correct_padding(self, *mocks):
+
+        response = self.client.post(
+            reverse("verzending-list"),
+            {
+                "betrokkene": "https://foo.com/persoonX",
+                "informatieobject": eio_url,
+                "aardRelatie": AfzenderTypes.geadresseerde,
+                "toelichting": "Verzending van XYZ",
+                "ontvangstdatum": (timezone.now() - timedelta(days=3)).strftime(
+                    "%Y-%m-%d"
+                ),
+                "verzenddatum": timezone.now().strftime("%Y-%m-%d"),
+                "contactPersoon": "https://foo.com/persoonY",
+                "contactpersoonnaam": "persoonY",
+                "binnenlandsCorrespondentieadres": {
+                    "huisletter": "Q",
+                    "huisnummer": 1,
+                    "huisnummerToevoeging": "XYZ",
+                    "naamOpenbareRuimte": "ParkY",
+                    "postcode": "1800XY",
+                    "woonplaatsnaam": "Alkmaar",
+                },
+                "buitenlandsCorrespondentieadres": {
+                    "adresBuitenland_1": "Adres 1",
+                    "adresBuitenland_2": "Adres 2",
+                    "adresBuitenland_3": "Adres 3",
+                    "landPostadres": "https://foo.com/landY",
+                },
+                "correspondentiePostadres": {
+                    "postBusOfAntwoordnummer": "1",
+                    "postadresPostcode": "3322DT",
+                    "postadresType": "antwoordnummer",
+                    "woonplaatsnaam": "4",
+                },
+            },
+        )
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(response.data["code"], "invalid-amount")
+
+    def test_validate_zero_address_on_create(self):
+        eio = EnkelvoudigInformatieObjectCanonicalFactory.create(
+            latest_version__creatiedatum="2018-12-24",
+            latest_version__informatieobjecttype=INFORMATIEOBJECTTYPE,
+        )
+
+        eio_url = reverse(
+            "enkelvoudiginformatieobject-detail",
+            kwargs={"uuid": eio.latest_version.uuid},
+        )
+
+        response = self.client.post(
+            reverse("verzending-list"),
+            {
+                "betrokkene": "https://foo.com/persoonX",
+                "informatieobject": eio_url,
+                "aardRelatie": AfzenderTypes.geadresseerde,
+                "toelichting": "Verzending van XYZ",
+                "ontvangstdatum": (timezone.now() - timedelta(days=3)).strftime(
+                    "%Y-%m-%d"
+                ),
+                "verzenddatum": timezone.now().strftime("%Y-%m-%d"),
+                "contactPersoon": "https://foo.com/persoonY",
+                "contactpersoonnaam": "persoonY",
+            },
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(response.data["code"], "invalid-amount")
+
+    def test_validate_one_address_on_partial_update(self):
```

Comment:
Ik zou of de naam van de test hier aanpassen of de test zelf. Als je het gedrag zoals in de test naam wilt testen klopt je test niet maar als je de `patch` wilt testen met een bestaand adres klopt de naam van je test niet.